curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc23991f86c76b9248c6eb8.md

```diff
@@ -9,6 +9,15 @@ dashedName: step-6
 
 In the previous step, you put the `h1`, `h2`, comment, and `p` elements inside the `main` element. This is called *nesting*. Nested elements should be placed two spaces further to the right of the element they are nested in. This spacing is called indentation and it is used to make HTML easier to read.
 
+Here is an example of nesting and indentation:
```

Comment:
step 6 is getting better but still pretty popular on the forum
https://forum.freecodecamp.org/search?q=step%206%20%20%23html-css%20cat%20photo%20app%20after%3A2024-03-01

so showing a code example should better direct campers on what proper indentation looks like IMO